From 845c8186cb4fce1578713119967e357ff8d94bc1 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Wed, 24 Jul 2024 12:44:43 +0300
Subject: [PATCH 13/17] compulab: Add imx93 family

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/mach-imx/imx9/Kconfig | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/mach-imx/imx9/Kconfig b/arch/arm/mach-imx/imx9/Kconfig
index 1f78b1b6a31..af5ef2b4289 100644
--- a/arch/arm/mach-imx/imx9/Kconfig
+++ b/arch/arm/mach-imx/imx9/Kconfig
@@ -68,6 +68,14 @@ config TARGET_IMX93_9X9_QSB
 	select IMX93
 	select IMX9_LPDDR4X
 
+config TARGET_MCM_IMX93
+	bool "mcm_imx93"
+	select IMX93
+
+config TARGET_UCM_IMX93
+	bool "ucm_imx93"
+	select IMX93
+
 config TARGET_IMX95_19X19_EVK
 	bool "imx95_19x19_evk"
 	select IMX95
@@ -90,6 +98,8 @@ source "board/variscite/imx93_var_som/Kconfig"
 source "board/freescale/imx93_qsb/Kconfig"
 source "board/freescale/imx95_evk/Kconfig"
 source "board/toradex/verdin-imx95/Kconfig"
+source "board/compulab/ucm-imx93/Kconfig"
+source "board/compulab/mcm-imx93/Kconfig"
 
 endif
 
-- 
2.34.1

