From 573f84d337debae6ed8de082e8bc50bfb9c906ab Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 6 Jul 2026 12:10:07 +0000
Subject: [PATCH] compulab: imx952: monitor_cmds: Improve lm command

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 sm/utilities/monitor/monitor_cmds.c | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/sm/utilities/monitor/monitor_cmds.c b/sm/utilities/monitor/monitor_cmds.c
index 3c6f683..bc3f45c 100755
--- a/sm/utilities/monitor/monitor_cmds.c
+++ b/sm/utilities/monitor/monitor_cmds.c
@@ -1388,6 +1388,7 @@ static int32_t MONITOR_CmdLm(int32_t argc, const char * const argv[])
                 break;
             case 1:  /* default */
                 s_lm = lm;
+		s_lm  = strtoul(argv[1], NULL, 0);
                 break;
             case 2:  /* boot */
                 status = LMM_SystemLmBoot(0U, 0U, lm, &g_swReason);
@@ -1420,7 +1421,11 @@ static int32_t MONITOR_CmdLm(int32_t argc, const char * const argv[])
     }
     else
     {
-        status = SM_ERR_MISSING_PARAMETERS;
+        for (uint8_t idx = 0U; idx < ARRAY_SIZE(cmds); idx++)
+        {
+            printf("lm %s\n", cmds[idx]);
+        }
+        status = SM_ERR_SUCCESS;
     }
 
     /* Return status */
-- 
2.43.0

