From 7ff86478d875892cedd04fa5476244b77f37cc92 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 16 Feb 2025 12:01:13 +0200
Subject: [PATCH 2/3] fsl_lpi2c: Prevent the SM null pointer exception

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 drivers/lpi2c/fsl_lpi2c.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/lpi2c/fsl_lpi2c.c b/drivers/lpi2c/fsl_lpi2c.c
index f9ffa3c..d5ae19f 100755
--- a/drivers/lpi2c/fsl_lpi2c.c
+++ b/drivers/lpi2c/fsl_lpi2c.c
@@ -977,6 +977,9 @@ status_t LPI2C_MasterTransferBlocking(LPI2C_Type *base, lpi2c_master_transfer_t
     uint16_t commandBuffer[7];
     uint32_t cmdCount = 0U;
 
+    if (base == 0x0)
+        return kStatus_InvalidArgument;
+
     /* Check transfer data size in read operation. */
     if ((transfer->direction == kLPI2C_Read) &&
         (transfer->dataSize > ((size_t)256 * (size_t)FSL_FEATURE_LPI2C_FIFO_SIZEn(base))))
-- 
2.34.1

