From d3c756e508bcc2a9e4d5f3edd0beb38e17d3abc7 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Wed, 6 Aug 2025 16:10:43 +0300
Subject: [PATCH] compulab: board mcimx95cpl: Use warm reset instead of wdog on
 B0

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 boards/mcimx95cpl/sm/brd_sm.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/boards/mcimx95cpl/sm/brd_sm.c b/boards/mcimx95cpl/sm/brd_sm.c
index 36b4b52..f528b59 100755
--- a/boards/mcimx95cpl/sm/brd_sm.c
+++ b/boards/mcimx95cpl/sm/brd_sm.c
@@ -455,9 +455,11 @@ void BRD_SM_ShutdownRecordSave(dev_sm_rst_rec_t shutdownRec)
 /*--------------------------------------------------------------------------*/
 /* Reset board                                                              */
 /*--------------------------------------------------------------------------*/
+#define DONT_USE_WDOG_ANY
 int32_t BRD_SM_SystemReset(void)
 {
     int32_t status = SM_ERR_SUCCESS;
+#ifndef DONT_USE_WDOG_ANY
     rgpio_pin_config_t gpioConfig =
     {
         kRGPIO_DigitalOutput,
@@ -471,6 +473,7 @@ int32_t BRD_SM_SystemReset(void)
     /* Wait for PMIC to react */
     SystemTimeDelay(1000U);
 
+#endif
     /* Fall back to warm reset of the device */
     status = DEV_SM_SystemReset();
 
-- 
2.34.1

