From 3769a63f0fa149a78d71fc84e79669bdf8cd682f Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Thu, 6 Mar 2025 10:13:26 +0200
Subject: [PATCH] monitor_cmds: lm: Allow set default lm number

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 sm/utilities/monitor/monitor_cmds.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sm/utilities/monitor/monitor_cmds.c b/sm/utilities/monitor/monitor_cmds.c
index 317a194..ed07d66 100755
--- a/sm/utilities/monitor/monitor_cmds.c
+++ b/sm/utilities/monitor/monitor_cmds.c
@@ -1103,7 +1103,7 @@ static int32_t MONITOR_CmdLm(int32_t argc, const char * const argv[])
                     status = MONITOR_CmdLmInfo(argc - arg, &argv[arg]);
                     break;
                 case 1:  /* default */
-                    s_lm = lm;
+                    s_lm  = strtoul(argv[1], NULL, 0);
                     break;
                 case 2:  /* boot */
                     status = LMM_SystemLmBoot(0U, 0U, lm, &g_swReason);
-- 
2.34.1

