From c43cc1ecd8c7f72753e9b19ac1f72366efea1d9e Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Wed, 18 Jun 2025 15:59:28 +0300
Subject: [PATCH] configs: mx95cpl: change LPTPM1 ownership

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 configs/mx95cpl.cfg | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configs/mx95cpl.cfg b/configs/mx95cpl.cfg
index 8c61288..e25d144 100755
--- a/configs/mx95cpl.cfg
+++ b/configs/mx95cpl.cfg
@@ -363,7 +363,7 @@ IRQSTEER_M7         OWNER
 LPIT1               OWNER
 LPTMR1              OWNER
 LPTMR2              OWNER
-LPTPM1              OWNER
+LPTPM1              ACCESS
 LPUART3             OWNER, test
 MSGINTR1            OWNER
 MSGINTR2            OWNER
@@ -747,6 +747,7 @@ LPSPI5              OWNER
 LPSPI6              OWNER
 LPSPI7              OWNER
 LPSPI8              OWNER
+LPTPM1              OWNER
 LPTPM2              OWNER
 LPTPM3              OWNER
 LPTPM4              OWNER
-- 
2.34.1

