From 02571c03babe80836fffc01a6a25cefd4f2d3871 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Thu, 8 May 2025 11:46:40 +0300
Subject: [PATCH 2/3] board: mx95lp5: Fix default DDR_CONFIG timing name

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 boards/mx95lp5/ddr/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/boards/mx95lp5/ddr/Makefile b/boards/mx95lp5/ddr/Makefile
index 8b3a0e3..f46a206 100644
--- a/boards/mx95lp5/ddr/Makefile
+++ b/boards/mx95lp5/ddr/Makefile
@@ -1,4 +1,4 @@
-DDR_CONFIG ?= MIMX95_LPDDR5_EVK_19X19_6400MTS_FW2024.09_timing
+DDR_CONFIG ?= lpddr5_timing
 OBJS  += $(OUT)/$(DDR_CONFIG).o
 
 VPATH += boards/mx95lp5/ddr
-- 
2.34.1

